isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -391,6 +391,69 @@ Der in `isy.security.roles-claim-name` konfigurierte Wert unterstützt keine Ver
 Es ist sicherzustellen, dass das konfigurierte Roles-Claim auch vom Identity Provider befüllt werden.
 ====
 
+
+[[konfiguration-von-multi-iam-uris]]
+=== Konfiguration von mehreren IAM-URIs
+Zur Konfiguration von mehreren IAM-URIs ist es erforderlich, die Konfigurationsproperties `issuer-uri` und `jwk-set-uri` entsprechend zu definieren und mit besonderem Property-Pfad zu gruppieren.
+
+Der bisherige Property-Pfad
+
+- `*spring*.security.oauth2.resourceserver.jwt` ist durch
+- `*isy*.security.oauth2.resourceserver.jwt.tenants.*<tenant-id>*` zu ersetzen,
+
+wobei die `tenant-id` einzig als Gruppierungskriterium für die beiden Properties `issuer-uri` und `jwk-set-uri` eingesetzt wird.
+Somit kann diese beispielsweise mit fachlich aussagekräftigen Namen, wie der Name der Geschäftsanwendung, des Kommunikationskanals oder der Netzwerkzone, versehen werden.
+
+Die nachstehende Konfigurationstabelle zeigt eine Liste von Property-Pfad-Beispielen, um mehrere IAM-URIs zu definieren.
+Aus Gründen der Übersichtlichkeit wird das Präfix `isy.security.oauth2.resourceserver.jwt` in der unten aufgeführten Tabelle weggelassen.
+
+[[table-property-path-examples-for-multi-tenancy]]
+.Property-Pfad-Beispiele für Multi IAM-URIs
```

Comment:
Ich würde hier weiterhin nicht zwischen "interner" und "externer" URI unterscheiden. Ich hatte ja in einem meiner letzten Kommentare schon einen Verbesserungsvorschlag kommentiert.